tests/unit/views/notifications/project_crew_notification.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Project Crew Notification
+  As a user,
+  I want to be notified project crew member updates.
+
+  Scenario Outline: Twoflower is added to a project
+    Given Rincewind and Twoflower are project crew in the project Expo 2010
```

Comment:
Given statement is incorrect. It should be:

```feature
Given Vetinari is an owner of the Ankh-Morpork organization
Given Vetinari is an editor and promoter of the Ankh-Morpork 2010 project
Given Rincewind is a promoter of Ankh-Morpork 2010
```

Second, replace the `Rincewind` fixture with `Vimes` and `Twoflower` with `Ridcully` (as per the source material in Discworld).